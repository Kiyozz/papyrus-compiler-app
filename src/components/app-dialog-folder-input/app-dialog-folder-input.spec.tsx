import { shallow } from 'enzyme'
import React from 'react'
import AppDialogFolderInput, { Props } from './app-dialog-folder-input'

describe('<AppDialogFolderInput>', () => {
  const render = () => {
    const props: Props = {
      id: '1',
      name: 'toto',
      value: 'tata',
      onChange: () => {}
    }

    return (
      <AppDialogFolderInput {...props} />
    )
  }

  it('renders correctly', () => {
    const component = (
      render()
    )

    expect(component).toBeTruthy()
  })

  it('matches snapshot', () => {
    const component = shallow((
      render()
    ))

    expect(component).toMatchSnapshot()
    expect(component).toHaveProp('onClick')
    expect(component).toContainExactlyOneMatchingElement('input')
  })
})
