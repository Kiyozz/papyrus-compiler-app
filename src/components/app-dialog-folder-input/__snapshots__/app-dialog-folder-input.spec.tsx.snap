// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppDialogFolderInput> matches snapshot 1`] = `
<div
  className="app-dialog-folder-input"
  onClick={[Function]}
>
  <input
    className="form-control"
    id="1"
    name="toto"
    onChange={[Function]}
    value="tata"
  />
</div>
`;
